src/validators/Slot/SlotCreateBodyValidator.ts

```diff
@@ -8,9 +10,13 @@ import { SlotCreateBody } from "../../types/Slot/SlotCreateRequest";
  * Validate slot create request body
  */
 export default class SlotCreateBodyValidator implements SlotCreateBody {
-  [x: string]: boolean;
+  [x: string]: boolean | number;
```

Comment:
It seems we'd want to remove this line entirely for the same reasons I explained in `RoomUpdateBodyValidator`. The idea of this syntax is to allow extra keys that we haven't thought of yet, there's no reason to make this match the properties we are explicitly defining below...

Reply:
Were you not able to remove this in the end?